src/apps/material/helper.ts

```diff
@@ -411,19 +411,29 @@ export const isParallelReservation = (manifestations: Manifestation[]) =>
   hasCorrectAccessType(AccessTypeCode.Physical, manifestations) &&
   !isArticle(manifestations);
 
-// Because we  need to exclude the branches that are blacklisted, we need to use a custom hook to prevent duplicate code
+// Because we  need to exclude the branches that are blacklisted, we need to
```

Comment:
Very very small nitty picky thing. Can we remove the extra space between "we" and "need"? 😄 
```suggestion
// Because we need to exclude the branches that are blacklisted, we need to
```